examples/rust-simple/README.md

```diff
@@ -8,9 +8,52 @@ It will
 - Every 10 seconds, write a log saying "Hi! I'm a Game Server"
 - After 60 seconds, call `SDK::Shutdown()` to shut the server down.
 
+## Running locally
```

Comment:
What are the `build` and `build-server` targets used for in the makefile if not for running locally? Can you document their use in the README (or in the makefile)?

Reply:
I added run locally with docker and without sections.

Reply:
Thanks!